### ISSUE TITLE
Incompatibility causing messy_cobweb to boot player out

### ISSUE THREAD

**Issue by ImWhiff** [sf-inc/sihywtcamd#55]
Mod version: 1.10.4+ 1.21 (latest as of 08/16/24)
Game version: Fabric 1.21 

When shielding a spider attack while webProjectileGoal is true, the game boots me to main menu. It doesn't when it is false.

- "Caused by: java.lang.IllegalArgumentException: Can't find id for 'Block{sihywtcamd:messy_cobweb}"
- "lost connection: Internal Exception: io.netty.handler.codec.EncoderException: Failed to encode packet 'clientbound/minecraft:block_update'"

log: [latest.log](https://github.com/user-attachments/files/16634321/latest.log)

**Comment by charlyb01** [comment]
Hello, thanks for the report! Mmh that's weird, I'll try this out. Just a quick thought, maybe it's because you ran the game without the option enabled and didn't restart the game in-between? This setting needs a game restart to be effective, you should get a warning when updating it.

**Comment by ImWhiff** [comment]
Nah, I restarted for sure. Both times when trying it out when on false and true.

**Comment by charlyb01** [comment]
I can't reproduce the issue. Are you on a server? If yes, did you update the config on the server and reboot it?

**Comment by charlyb01** [comment]
It shouldn't be an incompatibility issue. Make sure to not update the config during a fight with a spider because if a messy cobweb is spawned and then you restart the game, issues could happen

**Comment by ImWhiff** [comment]
Well it turns out it has nothing to do with blocking an attack. It just boots me out of the world when a spider hits me with its web. 

I ran a world with just the mod alone and didn't get booted out, but did on my normal modded world, so I am thinking it is an incompatibility. I did made sure to restart after config changes.

[latest.log](https://github.com/user-attachments/files/16673597/latest.log)

**Comment by charlyb01** [comment]
Thank you for the additional information. I have another issue with blocks in parallel, so I'll investigate those asap!

**Comment by charlyb01** [comment]
If you have the opportunity to test with less mods to determine what mods can cause your issue, that would be a great help :)

**Comment by seriousfreezing** [comment]
Same.
https://paste.atlauncher.com/view/66511b85-f6b9-41f5-bbc3-2b5ad1a0164d
[latest.log](https://github.com/user-attachments/files/17004695/latest.log)

**Comment by TinLikesCode** [comment]
i have a similar issue with this modset:

https://github.com/user-attachments/files/17004873/logs.and.specifications.txt

Im unsure if its an incompatibility, i just wanted to know if this issue has been resolved?

**Comment by charlyb01** [comment]
@TinLikesCode I didn't find anything related to the issue or to my mod in your log, but no this is not yet fixed.  
@seriousfreezing @ImWhiff You both have the same issue. Thanks to this issue #61, I discovered that your issue is an incompatibility with ModernFix. I don't know yet what exactly causes the issue, but at least now I know where to look. I will keep you tuned

**Comment by TinLikesCode** [comment]
> @TinLikesCode I didn't find anything related to the issue or to my mod in your log, but no this is not yet fixed.  
> 
> @seriousfreezing @ImWhiff You both have the same issue. Thanks to this issue #61, I discovered that your issue is an incompatibility with ModernFix. I don't know yet what exactly causes the issue, but at least now I know where to look. I will keep you tuned

Well thanks anyway, I _was_ able to find a temporary fix by disabling the attack using the config so I don't have a problem, and I'm pretty sure I am using modern fix
